lxc/list.go

```diff
@@ -663,6 +665,31 @@ func (c *cmdList) memoryUsageColumnData(cInfo api.InstanceFull) string {
 	return ""
 }
 
+func (c *cmdList) memoryUsagePercentColumnData(cInfo api.InstanceFull) string {
+
+	if cInfo.IsActive() && cInfo.State != nil && cInfo.State.Memory.Usage > 0 {
+
+		if cInfo.ExpandedConfig["limits.memory"] != "" {
```

Comment:
Just worth keeping in mind that this won't work for anyone using a memory limit in percents.

Reply:
I've added a check and for now return empty string if limit is in percent.

Elegant solution would be to calculate how much memory percentage represents and use that in the division below. I'm just not sure if that information is available in this function.